src/test/java/uk/gov/ons/ctp/response/collection/exercise/endpoint/CollectionExerciseExecutionEndpointUnitTests.java

```diff
@@ -72,8 +75,6 @@ public void requestSampleUnits() throws Exception {
     actions
         .andExpect(status().isOk())
         .andExpect(handler().handlerType(CollectionExerciseExecutionEndpoint.class))
-        .andExpect(handler().methodName("requestSampleUnits"))
-        .andExpect(jsonPath("$.*", hasSize(1)))
-        .andExpect(jsonPath("$.sampleUnitsTotal", is(SAMPLEUNITSTOTAL)));
+        .andExpect(handler().methodName("setReadyForLive"));
```

Comment:
The endpoint being invoked during this test does not return a json object, let alone a json object with a `sampleUnitTotal`. The endpoint being invoked activates the sample then simply returns an 'empty' 200
```
    sampleSummaryService.activateSamples(id);
    return ResponseEntity.ok().build();
```
Therefore I've removed the spurious failing assertion so it aligns with the implementation.